src/nav/AppContainer.js

```diff
@@ -11,11 +11,23 @@ import { registerAppActivity } from '../utils/activity';
 import { checkCompatibility } from '../api';
 import LoadingScreen from '../start/LoadingScreen';
 import CompatibilityScreen from '../start/CompatibilityScreen';
+import { Auth, Outbox, Actions } from '../types';
+import { attemptToSendMessages } from '../outbox/outboxMessageActions';
```

Comment:
Nit: It is a popular convention to call these functions 'tryDoSomething' so in that case `trySendMessages` would be the more conventional name (or `tryToSendMessages` but I think `try` is enough).